app/Models/Post.php

```diff
@@ -13,4 +13,9 @@ class Post extends Model
         'title',
         'content',
     ];
+
+    public function users()
```

Comment:
```suggestion
    public function user()
```
1-n thì user là số ít chứ nhỉ?

Reply:
e nhầm ạ, e đã sửa lại